playbooks/deployments.md

```diff
@@ -0,0 +1,137 @@
+
+#### Open and merge a PR to trigger a release
+
+E.g.:
+
+- via `hokusai pipeline promote` ([Volt](https://github.com/artsy/volt/blob/master/.circleci/config.yml), ...)
+- via `hokusai production deploy ...`
+  ([Kaws](https://github.com/artsy/kaws/blob/2e8e0ca8be2bc14e30e979cf2481e01767762f5e/.circleci/config.yml),
+  [Force](https://github.com/artsy/force/blob/4e81b8f92d40bcdcd6c575be31ac561e1500f203/.circleci/config.yml))
+- via `git push heroku master` ([Impulse](https://github.com/artsy/impulse/blob/master/.circleci/config.yml))
+
+This leverages the exact set-up already in use to deploy staging, and is independent of any developer's
+environment. The pull request also acts as a convenient record of the deploy, including commits and any discussion.
+It's possible, however, for deploys to end up blocked or delayed by other CI tasks. This approach also requires a
+little extra CircleCI configuration.
+
+Some projects trigger a deploy by PR-ing `master` to a `release` branch. This risks including commits that haven't
+completed a full CI run over the `master` branch. To avoid this, others update a `staging` or `master-stable`
+branch after the staging deploy is complete and PR from that branch.
+
+## Recommendations
+
+- It's acceptable for projects to start by recommending a basic `hokusai pipeline promote`
+- When setting up projects, use `hokusai setup` in combination with the templates in
+  [artsy/artsy-hokusai-templates](https://github.com/artsy/artsy-hokusai-templates). They provide reasonable
+  starting points for `.circleci/config.yml` and other configuration files.
+- Full-fledged projects should drive releases (via CircleCI) with pull requests from a `staging` branch to a
+  `release` branch as described above.
+  - The staging deploy must update a `staging` _branch_ (instead of publishing a `staging` tag) as a basis for
+    release PRs.
+  - Release PRs can usually be created from a URL like `<github project URL>/compare/release...staging?expand=1`.
+  - See [Reflection's .circleci/config.yml](https://github.com/artsy/reflection/blob/master/.circleci/config.yml)
+    as an example of a full set-up for staging and production deploys.
+  - Note that this requires
+    [configuring a read+write key for CircleCI](https://circleci.com/docs/2.0/gh-bb-integration/#creating-a-github-user-key)
+    rather than the default read-only key.
+  - Since a `hokusai pipeline promote` promotes the image currently in use by staging at the time of the command,
+    it's possible that doesn't match what was merged into the `release` branch by the PR. In the future, we'd like
+    to avoid this race condition with an argument to `hokusai pipeline promote`, possibly like
+    `--only $CIRCLE_SHA1`.
+
+In all cases:
+
+- Announce deploys for visibility in [#dev](https://artsy.slack.com/messages/dev)
+- Document accurate deploy procedure in README
+
+## Advanced topics
```

Comment:
Can we add rollbacks to here?

Reply:
Oh just had a thought-- maybe on head project's README we should _also_ describe how to roll back a deploy.

Reply:
Updated to mention roll-backs in hot-fixes.